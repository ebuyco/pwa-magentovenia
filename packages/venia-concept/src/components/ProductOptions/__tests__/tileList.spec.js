import React from 'react';
import testRenderer from 'react-test-renderer';

import TileList from '../tileList';

jest.mock('src/classify');

const defaultProps = {
    items: [
        {
            id: '1',
            label: 'foo'
        }
    ]
};

test('renders TileList component correctly', () => {
    const component = testRenderer.create(<TileList {...defaultProps} />);
    expect(component.toJSON()).toMatchSnapshot();
});
