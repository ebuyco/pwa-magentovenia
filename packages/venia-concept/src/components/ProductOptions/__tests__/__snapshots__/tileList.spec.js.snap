// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders TileList component correctly 1`] = `
<div
  className="root"
>
  <button
    className="root"
    onBlur={[Function]}
    onClick={[Function]}
    onFocus={[Function]}
  >
    <span>
      foo
    </span>
  </button>
</div>
`;
